nucypher/characters/control/specifications/alice.py

```diff
@@ -36,9 +39,26 @@ class PolicyBaseSchema(BaseSchema):
         load_only=True,
         click=click.option('--value', help="Total policy value (in Wei)", type=types.WEI))
 
+    rate = fields.Wei(
+        load_only=True,
+        required=False,
+        click=options.option_rate
+    )
+
     # output
     policy_encrypting_key = fields.Key(dump_only=True)
 
+    @validates_schema
```

Comment:
👍 - it's cool that all of the validation logic does not have to be done in one function.

Reply:
Continued scoping: Handle errors and error output via `Emitters`